src/classes/Shapes/BoxShape.js

```diff
@@ -79,24 +79,9 @@ Goblin.BoxShape.prototype.findSupportPoint = function( direction, support_point
 	]
 	*/
 
-	// Calculate the support point in the local frame
-	if ( direction.x < 0 ) {
-		support_point.x = -this.half_width;
-	} else {
-		support_point.x = this.half_width;
-	}
-
-	if ( direction.y < 0 ) {
-		support_point.y = -this.half_height;
-	} else {
-		support_point.y = this.half_height;
-	}
-
-	if ( direction.z < 0 ) {
-		support_point.z = -this.half_depth;
-	} else {
-		support_point.z = this.half_depth;
-	}
```

Comment:
This might have been an optimization as calling sign function should be slower

Reply:
This is going to be roughly the same in terms of performance, however, support point will be closer to actual contact point, which helps GJK converge faster.